packages/react-draggable-dialog/src/components/DraggableDialog/utils/getParsedDraggableMargin.test.ts

```diff
@@ -0,0 +1,232 @@
+import { getParsedDraggableMargin } from './getParsedDraggableMargin';
+import {
+  DraggableDialogMargin,
+  DraggableDialogMarginAxis,
+  DraggableDialogMarginViewport,
+} from '../DraggableDialog.types';
+
+describe('getParsedDraggableMargin', () => {
+  const defaultMargin = {
+    top: 0,
+    end: 0,
+    bottom: 0,
+    start: 0,
+  };
+
+  describe('when margin is undefined', () => {
+    it('should return default margin', () => {
+      const result = getParsedDraggableMargin(undefined);
+      expect(result).toEqual(defaultMargin);
+    });
+  });
+
+  describe('when margin is a number', () => {
+    it('should return uniform margins for various number types', () => {
```

Comment:
You could leverage the [it.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) and have one assertion per tests with a proper scenario title

```suggestion
  it.each(testCases, 'should return margins for number types: %o %o', (actual, expected) => {
```